index.html

```diff
@@ -1572,6 +1589,66 @@
 
 <aside class="issue">Some services have the user pay for their use in data. These services aren't necessarily retaliating by denying their services to users who refuse to pay with data, but the details are more complex than we've had time to write.</aside>
 
+## Support Choosing Which Information to Present {#support-choosing-info}
+
+<div class="practice">
+  <span class="practicelab" id="principle-support-choosing-info">
+    [=User agents=] should support [=people=] in choosing which information they provide to [=parties=] that
+    request it, up to and including allowing users to provide arbitrary information.
+  </span>
+</div>
+
+[=Parties=] can invest time and energy into automating ways of gathering [=data=] from [=people=] and can
+design their products in ways that make it a lot easier for [=people=] to disclose information than not, whereas
+[=people=] typically have to manually wade through options, repeated prompts, and deceptive patterns. In many
+cases, the absence of data — when a [=person=] refuses to provide some information — can also be identifying
+or revealing. Additionally, APIs can be defined or implemented in rigid ways that can prevent people from
+accessing useful functionality. For example, I might want to look for restaurants in a city I will be visiting
+this weekend, but if my geolocation is forcefully set to match my GPS, a restaurant-finding
+site might only allow searches in my current location.
+
+[=User agents=] should make it simple for [=people=] to [present the identity they wish
+to](#principle-identity-per-context) and to provide information about themselves or their devices in
+ways that they control. This empowers [=people=] to live in obscurity ([[?Lost-In-Crowd]],
+[[?Obscurity-By-Design]]) by rendering the provision of information according to their own
+preferences as easy as possible ([[?Obfuscation]]).
+
+<div class="practice">
+  <span class="practicelab" id="principle-no-facts-or-promises">
+    APIs should be designed such that data returned through an API does not assert a fact or make a
```

Comment:
```suggestion
    By default, APIs should be designed such that data returned through an API does not assert a fact or make a
```

Thinking this may highlight a nuance I was trying to call out with #191 originally. Which is to say the UA should assume that the user wants the more privacy preserving solution until the user decides to configure the UA to reveal more information about it.

Reply:
This text is aimed at the audience of designers of new APIs, so I think "by default" might get confused here (users may have default settings, but there's not as clear a concept of "default" in designing of new standards, except to give general advice, which is what "should be designed" means!).

Reply:
Ok - that makes sense and I agree this modification doesn't quite align with the intent then. Hmm I'll have to find a better place for this to go then.